src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,28 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
```

Comment:
Can `Carnivore` has this property? I think no, also task said that this property should be in `Herbivore` class